### DISCUSSION TITLE
Auto Clearing Docker stuff

### DISCUSSION THREAD

**Discussion by dbcaa** [pelican-dev/panel#488] [Ideas]
Maybe a docker button for `docker system prune`?

**Comment by rmartinoscar** [comment]
Could be a way to fix https://github.com/pelican-dev/panel/issues/250

**Comment by dbcaa** [comment]
also when you delete an egg delete the docker image associated with that egg

**Reply by iamkubi** [reply]
Many eggs share images, so it's not correct to do this.

**Comment by dbcaa** [comment]
i just cleared up 20 gb of docker images lol

**Comment by iamkubi** [comment]
The API is already there in wings (https://github.com/pelican-dev/wings/pull/6) just needs to be integrated into the node admin area.

**Reply by dbcaa** [reply]
Someone integrate it then lol @RMartinOscar

**Reply by rmartinoscar** [reply]
This only gets usage it doesn't run `docker prune`

**Reply by iamkubi** [reply]
Oh you're right, it's actually https://github.com/pelican-dev/wings/pull/7

**Reply by rmartinoscar** [reply]
Got it working but it doesn't return Which image was Untagged or Deleted and how much space was reclaimed so im keeping it for myself atm and will make a PR when it's clean enough and this gets converted into a _feature request issue_

**Comment by dbcaa** [comment]
![Screenshot 2024-07-16 at 7 09 50 PM](https://github.com/user-attachments/assets/6a49afca-9c8a-4578-be72-009adad856a0)

**Comment by Boy132** [comment]
It's planned, see https://github.com/pelican-dev/panel/issues/250 (https://github.com/pelican-dev/panel/discussions/54).
